app/src/org/commcare/android/logging/ReportingUtils.java

```diff
@@ -91,4 +94,14 @@ public static String getVersion() {
         }
     }
 
+    public static String getCommCareVersionString() {
```

Comment:
@wpride Yea it is, if you look at the implementation of that method, it returns a lot more information than just the version itself.